### PR TITLE
Only Admin Can Group Feedback

### PR DESCRIPTION
**Pull request by alexhoffman617**
@samblissard @cameronivey @bjsawyer  when someone has a chance to review this

### CODE REVIEW

File: src/app/components/group/group.component.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export class GroupComponent implements OnInit {
   @Input() groupId: string;
   @Input() messages: Message[];
   @Input() enabledGroup: string;
+  @Input() allowAdminFunctions; boolean = false;
```
Comment (samblissard) [inline src/app/components/group/group.component.ts]: You're not doing anything with this variable in the template of the group component itself. You should probably hide the edit group buttons if this is false.

Comment (samblissard) [inline src/app/components/group/group.component.ts]: Also, is this the right syntax? The ; after allowAdminFunctions seems like it should be a :

File: src/app/components/phase-steps/group-feedback/group-feedback.component.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
     <div fxLayout>
       <div fxFlex="50%">
         <h2>Groups</h2>
-        <md-input-container>
-          <input mdInput placeholder="Enter a new group name" [(ngModel)]="newGroupName" name="newGroupNameInput" (keyup.enter)="createNewGroup(newGroupName)"/>
+        <md-input-container *ngIf="allowAdminFunctionsLocal === true">
```
Comment (samblissard) [inline src/app/components/phase-steps/group-feedback/group-feedback.component.html]: Shouldn't this just be *ngIf="allowAdminFunctions"? I don't see an allowAdminFunctionsLocal variable in the group-feedback component.

**Comment by alexhoffman617** [comment]
I dont know what happened, I had some commit issues and soft rest the changes which was supposed to keep them all, but half of them got deleted and I didnt realize, readded the missing code I think

**Pull request merged**